js/app.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+var hours = [' 6 am ',' 7 am ',' 8 am ',' 9 am ',' 10 am ',' 11 am ' ,' 12 pm ',' 1 pm ',' 2 pm ',' 3 pm ',' 4 pm ',' 5 pm ',' 6 pm ',' 7 pm ',' 8 pm ', ' Total '];
+console.log(hours);
+
+var findRandomCookies = function (store) {
+  var total = 0;
+  var cookieSales = [];
+  //for loop of length of hours array to do some math
+  for (var i = 0; i < 15; i++) {
+    var sales = Math.floor(Math.random() * (store.maxCust - store.minCust + 1) + store.minCust);
```

Comment:
Your math is a little off here. You are correctly calculating the random number of customers per hour, but not the number of cookies sold. You will need to multiply this number by the average cookies sold per customer!